SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -7,6 +7,190 @@
 
 #include "SpatialGDKEditorSettings.generated.h"
 
+USTRUCT()
+struct FWorldLaunchSection
+{
+	GENERATED_BODY()
+
+	FWorldLaunchSection()
+		: Dimensions(2000, 2000)
+		, ChunkEdgeLengthMeters(50)
+		, StreamingQueryInterval(4)
+		, SnapshotWritePeriodSeconds(0)
+	{
+		LegacyFlags.Add(TEXT("bridge_qos_max_timeout"), TEXT("0"));
+		LegacyFlags.Add(TEXT("bridge_soft_handover_enabled"), TEXT("false"));
+		LegacyFlags.Add(TEXT("enable_chunk_interest"), TEXT("false"));
+	}
+
+	/** The size of the simulation in meters for the auto generated launch config. */
```

Comment:
There are a few places where the comments are ended with "for the auto generated launch config". Surely this applies to all the configs defined here? If so, I think it might be clearer to remove this everywhere